### PR TITLE
for secrets manager show SM title

### PR DESCRIPTION
**Pull request by cd-bitwarden**
## Type of change

```
- [ ] Bug fix
- [ x ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

The page title should change from Bitwarden Web Vault to Bitwarden Secrets Manager if they are inside of secrets manager
## Code changes

apps/web/src/app/core/router.service.ts - check if the url contains SM , if so show in the title Bitwarden Secrets Manager
apps/web/src/locales/en/messages.json - adding to messages for "Bitwarden Secrets Manager" and "Bitwarden Web Vault"

- **file.ext:** Description of what was changed and why

## Screenshots

image
## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/web/src/locales/en/messages.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "pageTitle": {
-    "message": "$APP_NAME$ Web Vault",
+    "message": "$APP_NAME$",
```
Comment (djsmith85) [inline apps/web/src/locales/en/messages.json:3]: The key should be changed if we changing context for the translators.

File: apps/web/src/app/core/router.service.ts [hunk 2]
```diff
@@ -23,7 +23,12 @@ export class RouterService {
       .subscribe((event: NavigationEnd) => {
         this.currentUrl = event.url;
 
-        let title = i18nService.t("pageTitle", "Bitwarden");
+        let title = i18nService.t("pageTitle", i18nService.t("bitWebVault"));
+
+        if (this.currentUrl.includes("/sm/")) {
+          title = i18nService.t("pageTitle", i18nService.t("bitSecretsManager"));
+        }
+
```
Comment (djsmith85) [inline apps/web/src/app/core/router.service.ts:31]: If this is the only place we use the `pageTitle` couldn't we just use `i18nService.t("bitWebVault")` and `i18nService.t("bitSecretsManager")` directly. Why use a placeholder if we have to have this conditional anyway

**Review by djsmith85 — CHANGES_REQUESTED** [review]

File: apps/web/src/app/core/router.service.ts (hunk 2 above)
Comment (cd-bitwarden) [reply apps/web/src/app/core/router.service.ts:31]: Yeah I agree, we can do it that way instead 👍

File: apps/web/src/locales/en/messages.json (hunk 1 above)
Comment (djsmith85) [reply apps/web/src/locales/en/messages.json:3]: @cd-bitwarden If the meaning of a message changes and we need to ensure that the translations are correct, then we need to change the key "pageTitle" and everywhere it is used. This essentially deletes all translations with key "pageTitle" and translators get a notification about a new string that needs translation.

If based on the other comment below, we can get rid of "pageTitle" even better

Comment (cd-bitwarden) [reply apps/web/src/locales/en/messages.json:3]: It looks like one place uses the pageTitle, so I reverted my changes to that message, and instead use the simple bitWebVault and bitSecretsManager instead 👍

**Review by djsmith85 — APPROVED** [review]
Changes are looking good.

**Comment by djsmith85** [comment]
@cd-bitwarden: Should we remove the `pagetitle` completely and replace it https://github.com/bitwarden/clients/blob/61d5b2df9c43c048c772c4d72c1c64a047a9b432/apps/web/src/app/layouts/navbar.component.html#L3 with `'bitWebVault'` ?

**Review by Thomas-Avery — APPROVED** [review]
Looks good.

**Review by djsmith85 — APPROVED** [review]

**Review by Hinton — APPROVED** [review]
Please add the jira ticket in the PR name.

**Pull request merged**